explorer/ast/value.cpp

```diff
@@ -512,7 +516,14 @@ void Value::Print(llvm::raw_ostream& out) const {
         }
         out << "]";
       }
-      out << fn_type.parameters() << " -> " << fn_type.return_type();
+      const auto* params = dyn_cast<TupleType>(&fn_type.parameters());
+      CARBON_CHECK(params) << "Parameters of function type must be a tuple type";
+      out << "(";
+      llvm::ListSeparator sep;
+      for (Nonnull<const Value*> element : params->elements()) {
+        out << sep << *element;
+      }
+      out << ") -> " << fn_type.return_type();
```

Comment:
While not required, the comma should be valid syntax. In the context of function types in particular, it's not obvious to me that we want to special-case the handling as a consequence. How much is this affecting?

Reply:
It just felt weird to me to print it like `fn (i32,)`. 

Only one test case is affected: `fail_impl_bad_member.carbon.test`